### DISCUSSION TITLE
FF7 crash in 7th heaven

### DISCUSSION THREAD

**Discussion by donlebun** [julianxhokaxhiu/FFNx#680] [Q&A] [answered]
Hello, I have a problem with ff7 in 7th Heaven
The game stops working
When I click play I get the following message: "The game crashed :("
At first I could enter the game but the message appeared as soon as I tried to drive the car, after that it took me out of the game. I did the same thing several times to see if one of them solved the problem until I eventually got to this situation.

**Comment by julianxhokaxhiu** [comment, marked as answer]
Feel free to use Discord for support. See https://github.com/julianxhokaxhiu/FFNx/discussions/160 cheers!
